api.js

```diff
@@ -1,24 +1,26 @@
 var https = require('https');
 var http = require('http');
 var zlib = require('zlib');
-module.exports = function (u, p, d, s, o) {
+module.exports = function (u, p, sId, d, s, o) {
```

Comment:
@pdfabbro This would be breaking change. Existing integration would now be passing database name into session id argument. Would fail after updating to latest. We could put at the end of arguments or maybe into options.

Reply:
ya sorry, I realized that after and updated it in another commit.  This has already been fixed